packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -259,28 +259,30 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
       onCreateLocalStreamView && onCreateLocalStreamView(localVideoViewOptions);
     }
     return (
-      <VideoTile
-        key={localParticipant.userId}
-        userId={localParticipant.userId}
-        renderElement={
-          localVideoStream?.renderElement ? (
-            <>
-              {
-                /* @conditional-compile-remove(call-with-chat-composite) @conditional-compile-remove(local-camera-switcher) */
-                localCameraCycleButton(localVideoCameraCycleButtonProps)
-              }
-              <StreamMedia videoStreamElement={localVideoStream.renderElement} />
-            </>
-          ) : undefined
-        }
-        showLabel={!(shouldFloatLocalVideo && isNarrow)}
-        displayName={isNarrow ? '' : strings.localVideoLabel}
-        initialsName={localParticipant.displayName}
-        styles={localVideoTileStylesThemed}
-        onRenderPlaceholder={onRenderAvatar}
-        isMuted={localParticipant.isMuted}
-        showMuteIndicator={showMuteIndicator}
-      />
+      <Stack tabIndex={0}>
```

Comment:
should this have a role or narration? (or what does narrator read out when it's focused on?)

Reply:
Good point, this should probably have a aria-label/role saying it has a drop down, will add that!

Reply:
If you are adding strings can you do that as a seperate PR, we may be too late to add new string changes to get the localized in time (this should be a sev2 or sev3 issue so not crucial to fix this)